release-notes/release-5.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: April 17, 2024
+---
+
+# Features
+
+- [Oracle Database Integration]() with default dashboard
+- [Apache Networking Integration]() with default dashboard
+- Automated log collection with the Vercel APM for __Build__, __Lambda__, __Static__, __Edge__, and __External__ immediately upon installation
+- Ability to parse attributes from non-standard JSON log messages
+- Improved sharing of RUM links with filters as query parameters within RUM to easily share user-specific sessions with your teammates
+- Users can set alerts based on their usage and budget on daily or monthly intervals
+- Advanced settings added to alerts. Users can configure alerts that will trigger when a certain specified threshold X is surpassed by Y number of occurrences over a given time interval Z
```

Comment:
Remove "Advnaced settings added to alerts." no need to call out advanced or not, just need to state what the feature is